plugins/modules/manageiq_group.py

```diff
@@ -276,7 +276,7 @@ def role(self, role_id, role_name):
                 if not role_res:
                     self.module.fail_json(msg="Role '%s' not found in manageiq" % role_name)
                 if len(role_res) > 1:
-                    self.module.fail_json(msg="Multiple roles found in manageiq with name '%s" % role_name)
+                    self.module.fail_json(msg="Multiple roles found in manageiq with name '%s'" % role_name)
```

Comment:
I am quite out of touch with python, but did you mean to introduce this single quote here?

Reply:
I think this change is intentional, since the closing single quote was missing.

(It's a bit more than a refactoring though, since it changes observable behavior.)

Reply:
ty ty, you are correct, I was having trouble seeing the other single quote. Nice work @russoz